user/enterprise/user-management-in-admin.md

```diff
@@ -0,0 +1,94 @@
+---
+title: User Management Commands
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise (TCIE) offers a new tab for admin users, the **User Activity** tab, where admin users can see active users and suspend or unsuspend specific users. This section describes how to obtain and use an Organizational-level token.
```

Comment:
Travis CI Enterprise (TCIE) offers a new tab for admin users ->

Travis CI Enterprise (TCIE) offers a new tab for admin users utilizing 'admin-v2' 


the reason: there are multipole tools in TCIE containing 'admin' in their names, we need to be specific where the feature has been added